.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,20 +2,52 @@
 name: Bug report
 about: Create a report to help us improve
 title: ''
-labels: 'Bug'
+labels: ''
 assignees: ''
 
 ---
 
-Please search the [existing issues](https://github.com/aws-amplify/amplify-android/search?q=%22your%20search%20query%20goes%20here%22) first.  If your issue is not reported yet, create one with the following information:
+Please search the [existing issues](https://github.com/aws-amplify/amplify-android/search?q=%22your%20search%20query%20goes%20here%22) first and confirm your issue is not reported yet. If not, create one with the following information:
 
-1. A **short** description
-2. The version of Amplify Android you're using
-3. The **code** you're using to call Amplify Android
-4. Any relevant [logs](https://github.com/aws-amplify/amplify-android/blob/main/CONTRIBUTING.md#getting-more-output)
-5. Any relevant guides or documentation you're referencing
-6. For API or DataStore issues, your `schema.graphql`
-7. For CLI issues, include any relevant logs in `~/.amplify/logs/amplify-cli-<date>.log`
+**Describe the bug**
+A clear and concise description of what the bug is.
 
-Please use [syntax highlighting](https://docs.github.com/en/free-pro-team@latest/github/writing-on-github/creating-and-highlighting-code-blocks#fenced-code-blocks) for any code/logs.
+**To Reproduce**
+_Example: steps to reproduce the behavior:_
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
 
+**Expected behavior**
+_Replace this with a clear and concise description of what you expected to happen._
+
+**Screenshots**
+_If applicable, add screenshots to help explain your problem._
+
+**Environment(please complete the following information):**
+ - Amplify Framework Version: [e.g. 1.17.3]
+ - Amplify Categories:
```

Comment:
This seems redundant with the labels that we add to issues.   Usually it's pretty obvious from the description which category they are asking about.